app/containers/BlockPage/selectors.js

```diff
@@ -7,15 +6,13 @@ const selectBlockPageDomain = state => state.get('blockPage', initialState);
 const makeSelectBlock = () =>
   createSelector(selectBlockPageDomain, globalState => {
     const block = globalState.get('block').toJS();
-
-    if (block.data.time) {
-      block.data.time = formatDate(block.data.time);
-    }
-
-    if (block.data.dividend_pay) {
-      block.data.dividend_pay =
-        block.data.dividend_pay === true ? 'true' : 'false';
-    }
+    block.prettyData = Object.assign(
```

Comment:
why you are not using object destructuring?

Reply:
changed